tools/ClassGen/NodeGen.cpp

```diff
@@ -111,8 +111,8 @@ int main(int argc, char **argv) {
       .addInput("BiasOffsets")
       .addMember(MemberType::VectorUnsigned, "Kernels")
       .addMember(MemberType::VectorUnsigned, "Strides")
-      .addMember(MemberType::VectorUnsigned, "Pads")
-      .addMember(MemberType::Unsigned, "Group")
+      .addMember(MemberType::VectorUnsigned, "Pads", /* addSetter */ true)
+      .addMember(MemberType::Unsigned, "Group", /* addSetter */ true)
```

Comment:
It doesn't seem like this is necessary, what's the motivation for this change?

Reply:
It is actually necessary. When splitting a node what happens is that a node is cloned multiple times. But in some cases the pads/group of the clone must be changed. This happens in `Conv2DSplitNodeModifier`. For example when splitting a Convolution with padding [1,1,1,1] both along the H/W axis, the convolution parts in the middle will have no padding so the padding of the clones must be modified.